scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -132,6 +151,8 @@ class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig)
     normal(tpe)
   }
 
+  private val fullEnv = Env(env.scopes :+ addedImportsScope)
```

Comment:
Ideally, I would expect that `addedImportsScope` is inserted right after the top-level import block (that's where the imports are going after the rewrite, right?). However, I wouldn't worry about getting everything 100% correct right now. Let's merge this as is and refine later if necessary.